otelcolbuilder/cmd/collector_windows.go

```diff
@@ -226,41 +195,3 @@ func withWindowsCore(elog *eventlog.Log) func(zapcore.Core) zapcore.Core {
 		return windowsEventLogCore{core, elog, zapcore.NewConsoleEncoder(encoderConfig)}
 	}
 }
-
-func updateSettingsUsingFlags(set *otelcol.CollectorSettings, flags *flag.FlagSet) error {
-	if set.ConfigProvider == nil {
-		resolverSet := &set.ConfigProviderSettings.ResolverSettings
-		configFlags := getConfigFlag(flags)
-
-		if len(configFlags) > 0 {
-			resolverSet.URIs = configFlags
-		}
-		if len(resolverSet.URIs) == 0 {
-			return errors.New("at least one config flag must be provided")
-		}
-		// Provide a default set of providers and converters if none have been specified.
-		// TODO: Remove this after CollectorSettings.ConfigProvider is removed and instead
-		// do it in the builder.
-		if len(resolverSet.ProviderFactories) == 0 && len(resolverSet.ConverterFactories) == 0 {
-			set.ConfigProviderSettings = newDefaultConfigProviderSettings(resolverSet.URIs)
-		}
-	}
-	return nil
-}
-
-func newDefaultConfigProviderSettings(uris []string) otelcol.ConfigProviderSettings {
```

Comment:
this can be removed, since the defaults are now in the builder `otelcol-builder.yaml`?

Reply:
Hm, now that you mention it, this shouldn't work because it doesn't go through command parsing... But it did  seem to work for me in manual tests. I'll re-check.